src/main/java/org/ow2/proactive/connector/iaas/cloud/provider/azure/AzureProvider.java

```diff
@@ -87,10 +87,9 @@
  * @since 01/03/17
  */
 @Component
+@Log4j2
 public abstract class AzureProvider implements CloudProvider {
 
-    private static final Logger logger = Logger.getLogger(AzureProvider.class);
-
     @Getter
     protected final String type = "azureAbstract";
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Make this final field static too. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1170)